pkg/maps/ctmap/ctmap.go

```diff
@@ -627,6 +620,21 @@ func PurgeOrphanNATEntries(ctMapTCP, ctMapAny *Map) *NatGCStats {
 				}
 			} else {
 				stats.IngressAlive += 1
+				stats.EgressAlive += 1
```

Comment:
This is because we assume that such ingress NAT entry always has a related egress NAT entry, correct? Why can't we handle this in the case below, once we visit that egress NAT entry?

Reply:
Right, we are already accounting for this in the case below, I should remove it :+1: